README.md

```diff
@@ -124,5 +235,5 @@ localhost/this-is-my-image   latest   7558393975c7   1 minute ago   84.4MB
 * [x] A caching mechanism
 * [ ] Configurable caching
 * [x] A fancy name!
-* [ ] You can build images with podman (needs Ansible connection plugin for podman)
+* [ ] You can build images with podman (Requires https://github.com/ansible/ansible/pull/47519)
```

Comment:
I don't see why this is desirable, as podman (in this case) is just another layer between ansible-bender and buildah. Since building with buildah already works, I wouldn't try to put another layer in between.

Reply:
It's not another layer, it will be a different layer. Basically people will be able to choose a backend. Right now it's just buildah, but in future it will be buildah/podman/docker.

Reply:
Podman *is* another layer, because if you choose podman that uses buildah internally (as far as I know)